samples/subsys/usb/dfu/README.rst

```diff
@@ -0,0 +1,83 @@
+.. _usb_dfu:
+
+USB DFU Sample Application
+##########################
+
+Overview
+********
+
+This sample app demonstrates use of a USB DFU Class driver provided
+by the Zephyr project.
+
+Requirements
+************
+
+This project requires an USB device driver. Currently, the USB DFU
+class provided by the Zephyr project depends on DFU image manager and
+partition layout. Refer to :ref:`flash_partitions` for details about
+partition layout. You SoC have to run MCUboot as stage 1 bootloader.
+The sample will be build as application for the MCUboot bootloader.
+
+Building and Testing
+********************
+
+Building and signing the application
+====================================
+
+Sample can be built in the usual way (see :ref:`build_an_application`
+for more details) and flashed with regular flash tools, but will need
+to be loaded at the offset of SLOT-0.
+
+To boot the sample, the samle images must be signed.
+There is a `scripts\imgtool.py` script in the MCUboot source tree that
+can be used to sign the image (see MCUboot documentation for more details):
+
+.. code-block:: console
+
+   ~/src/mcuboot/scripts/imgtool.py sign \
+           --key ~/src/mcuboot/root-rsa-2048.pem \
+           --header-size 0x200 \
+           --align 8 \
+           --version 1.2 \
+           --included-header \
+           ./zephyr/zephyr.bin \
+           signed-zephyr.bin
+
+Build and flash MCUboot bootloader for Zephyr project as it is described in
+the MCUboot documentation `docs/readme-zephyr.md`. Then build, sign and flash
```

Comment:
change to:

    the `Using MCUboot with Zephyr`_ documentation.  Then build, sign, and flash